client/src/components/ProjectList/ProjectListItem/ProjectListItem.jsx

```diff
@@ -1,19 +1,21 @@
-import React from 'react'
-import style from './ProjectListItem.module.scss'
+import React from 'react';
+import style from './ProjectListItem.module.scss';
 
-const ProjectListItem = props => {
-    return (
-        <>
-            <li className={style.item}>
-                <div><span className={style.id}>#{props.id}</span></div>
-                <div>${props.capital}</div>
-                <div>{props.start}</div>
-                <div>{props.finish}</div>
-                <div>{props.desc}</div>
-                <div>{props.name}</div>
-            </li>
-        </>
-    )
+const ProjectListItem = (props) => {
```

Comment:
Props can be destructured
```suggestion
const ProjectListItem = ({ id, capital, start, finish, desc, name }) => {
```